website/docs/r/project.html.markdown

```diff
@@ -19,6 +19,7 @@ resource "mongodbatlas_project" "test" {
   name   = "project-name"
   org_id = "<ORG_ID>"
   project_owner_id = "<OWNER_ACCOUNT_ID>"
+  with_default_alerts_settings = false
```

Comment:
We don't want everyone doing this by default so please remove from the example. Those that need this simply need the description you have below.

Reply:
already changed :D